### PR TITLE
Point to new buildsec org

### PR DESCRIPTION
**Pull request by bradbeck**

### CODE REVIEW

**Review by sudo-bmitch — DISMISSED** [review]

**Review by joshualucas84 — DISMISSED** [review]
LGTM!

**Review by rgreinho — DISMISSED** [review]
👍

File: ssf-words.txt [hunk 1]
```diff
@@ -78,7 +78,7 @@ taskrun
 Tekton
 tekton
 tektoncd
-thesecuresoftwarefactory
+buildsec
```
Comment (sudo-bmitch) [inline ssf-words.txt:81]: Does this list need to be sorted?

Comment (rgreinho) [reply ssf-words.txt:81]: No, it just makes it nicer to read/update.

**Review by pxp928 — APPROVED** [review]

**Review by sudo-bmitch — APPROVED** [review]

**Pull request merged**